### PR TITLE
Create promotion section

### PR DESCRIPTION
**Pull request by CBinh710**
# Result
## Mobile screen
image
## Desktop screen screen
image

### CODE REVIEW

File: practice-1/src/index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
+    <p class="promotion">Sign up and get 20% off to your first order. <a href="" class="promotion-sign-up">Sign Up Now</a></p>
```
Comment (hungle-agilityio) [inline practice-1/src/index.html]: @CBinh710 please add default for href attribute, also double check for make sure correct design.

File: practice-1/src/styles/layouts/container.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+body {
```
Comment (hungle-agilityio) [inline practice-1/src/styles/layouts/container.css]: @CBinh710 I think you should be build `container` for each sections, avoid set width/height for `body`
Link refer: https://getbootstrap.com/docs/5.0/layout/containers/

File: practice-1/src/styles/sections/promotion.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  font-size: var(--font-size-sm);
+}
+
+.promotion-sign-up {
+  color: var(--color-primary);
+  font-family: var(--font-family-primary);
+  font-weight: 400;
+}
+
+@media screen and (min-width: 768px) {
```
Comment (hungle-agilityio) [inline practice-1/src/styles/sections/promotion.css]: @CBinh710 please add comment for this, what is device you support.

Comment (hungle-agilityio) [reply practice-1/src/styles/sections/promotion.css]: Please make sure implement for mobile first

File: practice-1/src/styles/sections/promotion.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+.promotion {
```
Comment (hungle-agilityio) [inline practice-1/src/styles/sections/promotion.css:1]: For each block styles, you might consider reorganizing the properties.
Link refer: https://9elements.com/css-rule-order/

Comment (CBinh710) [reply practice-1/src/styles/sections/promotion.css:1]: I updated it, thanks

File: practice-1/src/index.html (hunk 1 above)
Comment (CBinh710) [reply practice-1/src/index.html]: I add it, thanks

File: practice-1/src/styles/layouts/container.css (hunk 2 above)
Comment (CBinh710) [reply practice-1/src/styles/layouts/container.css]: I understood, please check when you can

File: practice-1/src/styles/sections/promotion.css (hunk 3 above)
Comment (CBinh710) [reply practice-1/src/styles/sections/promotion.css]: I made it, thanks

**Pull request merged**